### DISCUSSION TITLE
Sponsor Option 404 Screen

### DISCUSSION THREAD

**Discussion by Cameron-Fisher-506** [community/community#30088] [Other Feature Feedback, Questions, & Ideas]
Hi 

I deleted my Sponsor account to create a new one but now when I click on Sponsor option I get a 404 screen.

Can someone please help me?

Thanks

**Comment by jfullstackdev** [comment]
I'm not very sure of that but maybe this 

https://support.github.com/contact/account

posting it here also increases the chance of more visibility to Staff members, although there is no assurance too

**Reply by Cameron-Fisher-506** [reply]
Thank you @jdevstatic
